administrator/components/com_associations/associations.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+JHtml::_('behavior.tabstate');
+
+if (!JFactory::getUser()->authorise('core.manage', 'com_associations'))
+{
+	return JError::raiseWarning(404, JText::_('JERROR_ALERTNOAUTHOR'));
+}
+
+JLoader::register('AssociationsHelper', __DIR__ . '/helpers/associations.php');
```

Comment:
hum .... let's leave it for now. we will check later if the helper is really needed.